hapic/ext/bottle/context.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         app: bottle.Bottle,
         default_error_builder: ErrorBuilderInterface=None,
     ):
+        self._handled_exception_and_http_codes = []  # type: typing.List[typing.Tuple[typing.Type[Exception], int]]  # nopep8
```

Comment:
With a `HandledExcetion` class, instead of 

```
# type: typing.List[typing.Tuple[typing.Type[Exception], int]]  # nopep8
```

typing will become: 

```
# type: typing.List[HandledException]
```

Reply:
done